packages/lending/src/protocol.utils.ts

```diff
@@ -125,3 +125,27 @@ export const removePortfolioDynamicFields = (portfolio: any) => {
     });
   }
 };
+
+export const getLstApyFromMap = (address: string, lstTokenAPYMap: Record<string, string>) => {
```

Comment:
can we change `address` to `tokenAddress` to be clearer?

Reply:
changed